test/parallel/test-runner-run.mjs

```diff
@@ -513,6 +520,33 @@ describe('require(\'node:test\').run', { concurrency: true }, () => {
     for await (const _ of stream);
     assert.match(stderr, /Warning: node:test run\(\) is being called recursively/);
   });
+
+  it('should run with different cwd', async () => {
```

Comment:
Can you add a test for a non-existent CWD? What happens?

Reply:
I just pushed a commit that adds a validation to check that the given `cwd` is a valid path and that it's a directory